app/src/org/commcare/views/media/AudioPlaybackButton.java

```diff
@@ -260,4 +294,36 @@ private void pausePlaying() {
         currentState = MediaState.Paused;
         refreshAppearance();
     }
+
+    private void animateProgress(int milliPosition, int milliDuration) {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.HONEYCOMB) {
+            ProgressBar progressBar = (ProgressBar)findViewById(R.id.circular_progress_bar);
+            int startPosition = 0;
+            int progressDuration = 500;
```

Comment:
From what I understand `ObjectAnimator.ofInt(progressBar, "progress", x, y);` uses reflection to animate the `progress` property of the `ProgressBar` argument. The last two integer arguments (`x`/`y`) specify the start/end range for that property. I would expect the larger the range, the finer the granularity of the animation. 
`setDuration` sets how long the animation should take to get from the start position to the end position. In this case, the animation duration should correspond to the audio playback time, which is how it is now.
The `ObjectAnimator` documentation is sort of bad, so not positive...